HRTool/src/main/webapp/app/app.js

```diff
@@ -5,20 +5,24 @@
 
 var app = angular.module('app', ['ngRoute', 'ngMaterial', 'ng', 'flow']);

```

Comment:
Do not use ngRoute. Use ui.router.
What is 'ng'? ng prefix should be reserved for official angular modules.